core/src/main/scala/org/apache/spark/sql/delta/commands/UpdateCommand.scala

```diff
@@ -22,18 +22,18 @@ import org.apache.spark.sql.delta.actions.{AddCDCFile, AddFile, FileAction}
 import org.apache.spark.sql.delta.commands.cdc.CDCReader.{CDC_TYPE_COLUMN_NAME, CDC_TYPE_NOT_CDC, CDC_TYPE_UPDATE_POSTIMAGE, CDC_TYPE_UPDATE_PREIMAGE}
 import org.apache.spark.sql.delta.files.{TahoeBatchFileIndex, TahoeFileIndex}
 import org.apache.hadoop.fs.Path
-
```

Comment:
nit: we have blank lines separating our import groups, so the above group is `other` and the below group is `org.apache.spark`. So add back the blank line please.